specification/contosowidgetmanager/Contoso.WidgetManager/tspconfig.yaml

```diff
@@ -17,22 +13,22 @@ options:
     emitter-output-dir: "{project-root}/.."
     examples-directory: "./examples"
     output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/widgets.json"
-  "@azure-tools/typespec-python":
-    emitter-output-dir: "{python-sdk-folder}/sdk/{service-directory-name}/{package-name}"
+  "@azure-tools/typespec-python":    
+    package-dir: "azure-contoso-widgetmanager"
     package-mode: "dataplane"
-    package-name: "azure-contoso-widgetmanager"
+    package-name: "{package-dir}"    
```

Comment:
@lmazuel can we have this package-name property defaulted to {package-dir} so we don't have to specify it in every config. Similar do we need to set the package-mode?

Reply:
@msyyc approved this change.